lib/power-after-fail-lib.robot

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Documentation       Collection of keywords related to the Power State After
+...                 Power Fail option
+
+
+*** Keywords ***
+Enter Power Management Options
```

Comment:
We do not need this kwd. Simply use `Enter Dasharo Submenu` as here: https://github.com/Dasharo/open-source-firmware-validation/blob/rework-terminal-kwds/dasharo-security/me-neuter.robot#L49